src/common/screen.rs

```diff
@@ -303,4 +310,9 @@ impl Screen {
             }
         }
     }
+    pub fn change_input_mode(&mut self, input_mode: InputMode) {
+        self.input_mode = input_mode;
+        self.get_active_tab_mut().unwrap().input_mode = self.input_mode;
```

Comment:
Shouldn't this be changed in all tabs and not just the active one?

Reply:
Probably should, but I haven't noticed any weird behaviors for user, because when he is in the tab mode and switches between them it still updates everything, but I can of course change that to update all tabs

Reply:
I think that just happens to work right now because the tab mode color is white.

Reply:
hmm, could be